README.md

```diff
@@ -53,7 +53,9 @@ As per [running the application](#running_the_application).
 3. Get busy
 
 ### Tests
-Pull requests must include tests for their proposed fixes/functionality. We use [JUnit](http://junit.org/) to write our tests and you can find the tests themselves [here](klarna-on-demand/src/androidTest/java/com/klarna/ondemand).
+Pull requests must include tests for their proposed fixes/functionality. We use [JUnit](http://junit.org/) to write our tests and you can find the tests themselves [here](klarna-on-demand/src/test/java/com/klarna/ondemand).
+
+If you're using Android Studio, use the [following guide](https://github.com/klarna/klarna-on-demand-android/blob/master/doc/android_studio_setup.md) in order to setup a JUnit testing environment.
```

Comment:
you should use relative links, as in `[following guide](doc/android_studio_setup.md)`.

Seems I've also sinned, and you can probably change the link on line 56 to `src/test/java/com/klarna/ondemand`.

Finally, I approve of Yuval's notion, but drop the "on Android studio" at the end as we already says this only applies to Android Studio.